plugin.program.mceremote/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.program.mceremote"
+       name="MCERemote"
+       version="3.0.0"
+       provider-name="John Rennie">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>executable</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>windx</platform>
+    <language>en</language>
```

Comment:
this does not apply, the addon does not provide content (video, music, etc)

Reply:
Can you clarify what the problem is? Is it the platform tag that should be changed?

Reply:
The language one, it refers to content. Your add-on just configures a remote

Reply:
Please remove it

Reply:
Done!